### PR TITLE
Use npm 7

### PR DESCRIPTION
**Pull request by artonge**
:warning:  Required checks need to be updated.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 module.exports = {
 	extends: [
 		'@nextcloud',
-	]
+	],
+	rules: {
+		// Allow shallow import of @vue/test-utils in order to be able to use it in
+		// the src folder
```
Comment (skjnldsv) [inline .eslintrc.js:7]: Why is this needed in the src folder? :thinking:

Comment (skjnldsv) [reply .eslintrc.js:7]: Ah, src/tests?
You can create another eslintrc there then, that should extend or supersed this config :wink:

File: package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
 		"test:update-snapshots": "NODE_ENV=test jest --updateSnapshot"
 	},
 	"engines": {
-		"node": ">=10.0.0"
+		"node": ">=14.0.0"
```
Comment (skjnldsv) [inline package.json:31]:
```suggestion
		"node": ">=14.0.0",
		"npm": ">=7.0.0"
```

**Review by skjnldsv — APPROVED** [review]
Small nitpicks, otherwise good! :)

File: .eslintrc.js (hunk 1 above)
Comment (artonge) [reply .eslintrc.js:7]: My plan was to make a PR to add it to the shared eslint config as it already contains an exception for `@vue/test-utils`.
Is it the same use case ? https://github.com/nextcloud/eslint-config/blob/master/index.js#L97

**Review by nickvergessen — APPROVED** [review]
Seems to work

File: .eslintrc.js (hunk 1 above)
Comment (skjnldsv) [reply .eslintrc.js:7]: :scream:

Comment (skjnldsv) [reply .eslintrc.js:7]: https://github.com/nextcloud/eslint-config/pull/26 .........

Comment (artonge) [reply .eslintrc.js:7]: Not sure if that's a yes or a no :thinking:

Comment (skjnldsv) [reply .eslintrc.js:7]: I did not see that pass by, I'm not sure we should have done that in the first place, but yeah :shrug:
